networking/aws_load_balancer_operator/adding-tls-termination-and-ddos-protection.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/add-tls-termination-and-ddos-protection.adoc
+
+:_content-type: PROCEDURE
+[id="nw-adding-tls-termination-and-ddos-protection_{context}"]
+= Adding the tls termination and ddos-protection
+
+You can route traffic for the domain example.com to pods of Service example-service and add TLS termination and DDoS protection on the load balancer.
+
+.Prerequisites
+
+.Procedure
+
+. Deploy the operator and create an instance of the resource `AWSLoadBalancerController`:
++
+[source,yaml]
+----
+kind: AWSLoadBalancerController
+group: networking.olm.openshift.io/v1alpha1 <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: auto <3>
+  ingressClass: tls-termination <4>
+----
+
+<1> Defines the resource group of the `AWSLoadBalancerControlller` resource.
+<2> Defines name of a cluster.
```

Comment:
Not the name of the cluster, but the instance of the `aws-load-balancer-controller`.

Reply:
Addressed